src/typegoose.module.spec.ts

```diff
@@ -129,7 +129,7 @@ describe('TypegooseModule', () => {
     it('should return module that createdTypegooseProviders with provided connectionName', () => {
       const connectionName = 'OtherMongoDB';
 
-      const module = TypegooseModule.forFeature(models, connectionName);
+      const module = TypegooseModule.forFeature(models, connectionName); // is this wanted?
```

Comment:
is this wanted to be run? when yes, the value could be removed because it is unused

Reply:
It is unnecessary. I think removing it would be best.

Reply:
just to confirm, the whole line, or just the value assignment?